boards/st/nucleo_n657x0_q/doc/index.rst

```diff
@@ -199,33 +199,38 @@ First, connect the NUCLEO-N657X0-Q to your host computer using the ST-Link USB p
          .. note::
```

Comment:
For a more logical description of the steps:
We may want to split the note in 2: One part before the builld/Flash instruction, one part after.
Or put everything before.